src/main/scala/nl/knaw/dans/bag/Deposit.scala

```diff
@@ -351,6 +356,18 @@ object Deposit {
 
   val depositPropertiesName = "deposit.properties"
 
+  /**
+   * Creates a new Deposit with an empty Bag inside.
```

Comment:
(for all these comments, they also apply to `createFromData` and other documentation below)

1. please format the docs such that they will line out nicely
2. usually the `@param` lines don't start their documentation with a capital letter; just use lowercase letters
3. "_if none provided SHA1 is used._" --> "_If none are provided, SHA-1 is used._"
4. use backticks for the `state.label`, `depositor.userId` and `bagId`
5. use hyperlink syntax for 'nl.knaw.dans.bag.Deposit'